test/encryption-decryption.test.ts

```diff
@@ -59,6 +66,33 @@ describe('aes-256-gcm', () => {
     }
   });
 
+  it(`can encrypt/decrypt bytes with AES-GCM Encryption and latest serialization version`, async (done) => {
+    try {
+      const key = generateRandomKey();
+      const data = utf8ToBytes(
+        'this is a test 这是一个测试 이것은 테스트입니다 これすهذا اختبار यह एक परीक्षण है Это проверка ഇതൊരു പരീക്ഷ'
+      );
+
+      const strategy = CipherStrategy.AES_GCM;
+
+      const result = await encryptWithKey({ key, data, strategy });
+
+      if (result.serialized === null) {
+        throw new Error('serialized should not be null here');
+      }
+      const decryptedWithSourceKey = await decryptWithKey({
+        serialized: result.serialized,
+        key,
+      });
+
+      expect(bytesToUtf8(decryptedWithSourceKey as Uint8Array)).toEqual(bytesToUtf8(data));
```

Comment:
This typecast seems unnecessary. The result of `decryptedWithSourceKey` should be `Promise<Uint8Array>`  so if the typecast is required here the typings are incorrect.